### PR TITLE
Chapter 16-01 16_1_Accediendo_a_Bitcoind_en_C_con_las_Bibliotecas_RPC.md

### PR DESCRIPTION
**Pull request by javiervargas**

### CODE REVIEW

File: es/16_1_Accediendo_a_Bitcoind_en_C_con_las_Bibliotecas_RPC.md [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+```
+$ PATH="/sbin:$PATH"
+```
+Para un sistema Ubuntu, también querrá ajustar el `INSTALL_LIBPATH` en el fichero `Makefile` de `libbitcoinrpc` para instalar en `/usr/lib` lugar de `/usr/local/lib`:
+```
+$ emacs ~/libbitcoinrpc/Makefile 
+...
+INSTALL_LIBPATH    := $(INSTALL_PREFIX)/usr/lib
+```
+(Si prefiere mancillar su `/usr/lib`, la alternativa es cambiar su `etc/ld.so.conf` o sus archivos dependientes de manera apropiada  ... pero para una configuración de prueba en una máquina de prueba, esto probablemente esté bien)
```
Comment (icculp) [inline es/16_1_Accediendo_a_Bitcoind_en_C_con_las_Bibliotecas_RPC.md:42]: Si prefiere no mancillar

File: es/16_1_Accediendo_a_Bitcoind_en_C_con_las_Bibliotecas_RPC.md [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+Si `rpc_client` se inicializa correctamente, podrá enviar comandos RPC.
+
+Más tarde, cuando haya terminado con su conexión a `bitcoind`, debe cerrarla:
+```
+bitcoinrpc_global_cleanup();
+```
+
+### Pruebe el código de prueba
+
+El código de prueba se puede encontrar en el directorio src [15_1_testbitcoin.c ](src/15_1_testbitcoin.c). Descárguelo en su máquina de testnet, luego inserte la contraseña de RPC correcta (y cambie el usuario de RPC si no creó su servidor con StandUp).
```
Comment (icculp) [inline es/16_1_Accediendo_a_Bitcoind_en_C_con_las_Bibliotecas_RPC.md:151]: [15_1_testbitcoin.c] (remove space)
../src/15_1_testbitcoin.c

File: es/16_1_Accediendo_a_Bitcoind_en_C_con_las_Bibliotecas_RPC.md [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+int blocks;
+blocks = json_integer_value(jsonblocks);
+printf("Block Count: %d\n",blocks);
+```
+
+> :warning: **ADVERTENCIA:** Es extremadamente fácil segmentar su código C cuando se trabaja con objetos `jansson` si se confunde con el tipo de objeto que está recuperando. Haga un uso cuidadoso de `bitcoin-cli help` para saber lo que debe esperar, y si experimenta una falla de segmentación, primero observe sus funciones de recuperación de JSON.
+
+### Prueba el código de información
+
+Recupere el código de prueba del [directorio src](15_1_getmininginfo.c).
```
Comment (icculp) [inline es/16_1_Accediendo_a_Bitcoind_en_C_con_las_Bibliotecas_RPC.md:217]: ../src/15_1_getmininginfo.c

**Review by icculp — CHANGES_REQUESTED** [review]

File: es/16_1_Accediendo_a_Bitcoind_en_C_con_las_Bibliotecas_RPC.md [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+int blocks;
+blocks = json_integer_value(jsonblocks);
+printf("Block Count: %d\n",blocks);
+```
+
+> :warning: **ADVERTENCIA:** Es extremadamente fácil segmentar su código C cuando se trabaja con objetos `jansson` si se confunde con el tipo de objeto que está recuperando. Haga un uso cuidadoso de `bitcoin-cli help` para saber lo que debe esperar, y si experimenta una falla de segmentación, primero observe sus funciones de recuperación de JSON.
+
+### Prueba el código de información
+
+Recupere el código de prueba del [directorio src](../15_1_getmininginfo.c).
```
Comment (icculp) [inline es/16_1_Accediendo_a_Bitcoind_en_C_con_las_Bibliotecas_RPC.md:217]: ../src/15_1_getmininginfo.c

**Review by icculp — CHANGES_REQUESTED** [review]

**Review by icculp — APPROVED** [review]

**Comment by icculp** [comment]
@shannona ready to be merged

**Comment by shannona** [comment]
Merging.

**Pull request merged**